### PR TITLE
change license header to conform architecture guidelines

### PR DESCRIPTION
**Pull request by tmstff**
change license header to conform architecture guidelines

wait for https://github.com/greenbone/qm-header-templates/pull/2 / https://jira.greenbone.net/browse/OD-117 before merging

**Checklist**:

- [n/a ] Tests

### CODE REVIEW

**Review by bjoernricks — COMMENTED** [review]
IMHO the license header should always be kept. see https://www.gnu.org/licenses/gpl-howto.en.html

File: hello.go [hunk 1]
```diff
@@ -1,20 +1,8 @@
-/* Copyright (C) 2021 Greenbone Networks GmbH
+/* Copyright (C) Greenbone Networks GmbH
```
Comment (y0urself) [inline hello.go:1]:
```suggestion
/* Copyright (C) 2024 Greenbone Networks GmbH
```

File: hello_test.go [hunk 2]
```diff
@@ -1,19 +1,6 @@
-/* Copyright (C) 2021 Greenbone Networks GmbH
+/* Copyright (C) Greenbone Networks GmbH
```
Comment (y0urself) [inline hello_test.go:1]:
```suggestion
/* Copyright (C) 2024 Greenbone Networks GmbH
```

File: main.go [hunk 3]
```diff
@@ -1,20 +1,8 @@
-/* Copyright (C) 2021 Greenbone Networks GmbH
+/* Copyright (C) Greenbone Networks GmbH
```
Comment (y0urself) [inline main.go:1]:
```suggestion
/* Copyright (C) 2024 Greenbone Networks GmbH
```

**Review by y0urself — APPROVED** [review]

**Pull request merged**